recipes/xsv/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.13.0" %}
+
+package:
+  name: xsv
+  version: {{ version }}
+
+source:
+  url: https://github.com/BurntSushi/xsv/archive/{{ version }}.tar.gz
+  sha256: 2b75309b764c9f2f3fdc1dd31eeea5a74498f7da21ae757b3ffd6fd537ec5345
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Any explicit reason to skip windows? I see that the source repo is running on appveyor.

Reply:
for some reason I though rust was not available on windows, I see it is, I will give it a try.